common_utils/src/main/java/com/storyous/commonutils/Toaster.kt

```diff
@@ -0,0 +1,251 @@
+package com.storyous.commonutils
+
+import android.annotation.SuppressLint
+import android.content.Context
+import android.content.ContextWrapper
+import android.content.res.Resources
+import android.os.Build
+import android.view.Display
+import android.view.View
+import android.view.ViewGroup
+import android.view.WindowManager
+import android.widget.Toast
+import androidx.annotation.StringRes
+import timber.log.Timber
+
+object Toaster {
+
+    @JvmStatic
+    @JvmOverloads
+    fun showLong(context: Context, resId: Int, applyBlock: ((Toast) -> Unit) = {}): Toast {
```

Comment:
Neříkal bych tomu už `showLong` a `showShort`, když tam není ta `show` metoda.

Reply:
tam se ale vola show. nebo jsem to spatne pochopil?

Reply:
Jasně, já blbnu. To je špatně při tom použití. Třeba `DeliveryDetailFragment.kt` voláš `Toaster.showShort(...).show()`.

Reply:
🤦‍♂️ dik